roles/kubevirt-apb/templates/DaemonSet_libvirt.yml.j2

```diff
@@ -0,0 +1,74 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    kubevirt.io: libvirt
+  name: libvirt
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      labels:
+        kubevirt.io: libvirt
+      name: libvirt
+    spec:
+      containers:
+      - command:
+        - /libvirtd.sh
+        image: kubevirt/libvirt-kubevirt:v0.2.0
```

Comment:
This should be ```kubevirt/libvirt-kubevirt:{{ tag }}``` now that it's a template. Same for everywhere else.

Reply:
yes, i had it in mind ( but had dinner :) ). i d like to stick with a single tag for the moment, because that's how the upstream templates currently are like, a single tag for all kubevirt component